src/commands/data/schemas/util/metaschema.json

```diff
@@ -232,24 +233,214 @@
+        "required": {
+          "type": "array",
+          "items": { "type": "string" }
+        }
+      }
+    },
+    "documentCondition": {
+      "type": "object",
+      "properties": {
+        "configuration": { "$ref": "#/definitions/configurations" },
+        "type": {
+          "type": "string",
+          "enum": ["input"]
+        },
+        "required": {
+          "type": "array",
+          "items": { "type": "string" }
+        }
+      }
+    },
+    "initiatorHasRelationToUserInDataCondition": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "initiatorHasRelationToUserInData"
+          ]
+        },
+        "userIdField": {
+          "type": "string"
+        },
+        "relation": {
+          "type": "string",
+          "enum": ["isStaffOfTargetPatient"]
+        }
+      },
+      "required": [
+        "type"
+      ]
+    },
+    "initiatorHasRelationToGroupInData": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "initiatorHasRelationToGroupInData"
+          ]
+        },
+        "groupIdField": {
+          "type": "string"
+        },
+        "relation": {
+          "type": "string",
+          "enum": ["staff", "patient"]
+        },
+        "requiredPermission": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "type"
+      ]
+    },
+    "objectConfiguration": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["object"]
+        },
+        "required": {
+          "type": "array",
+          "items": {
             "type": "string"
           }
         },
-        "required": [
-          "type"
-        ]
+        "queryable": {
+          "type": "boolean"
+        }
+      },
+      "additionalProperties": true,
+      "required": ["type"]
+    },
+    "arrayConfiguration": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["array"]
+        },
+        "items": {
+          "type": "object"
+        },
+        "minItems": {
+          "type": "number"
+        },
+        "maxItems": {
+          "type": "number"
+        },
+        "contains": {
+          "type": "object"
+        },
+        "queryable": {
+          "type": "boolean"
+        }
+      }
+    },
+    "stringConfigurations": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["string"]
+        },
+        "minLength": {
+          "type": "number"
+        },
+        "maxLength": {
+          "type": "number"
+        },
+        "enum": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "pattern": {
+          "type": "string"
+        },
+        "required": {
```

Comment:
I don't believe this property exists for strings 🤔

Reply:
Correct! Will remove the property